<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cube2dWindow</class>
 <widget class="QMainWindow" name="Cube2dWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="cube2DGraphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="leftPushBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upPushBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightPushBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUseArrowsCube">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Arrows To Rotate Cube</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Rubik's Cube View in 3D</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
